doc/client_lua_api.md

```diff
@@ -621,6 +628,8 @@ Minetest namespace reference
   version entirely. To check for the presence of engine features, test
   whether the functions exported by the wanted features exist. For example:
   `if minetest.nodeupdate then ... end`.
+* `minetest.get_builtin_path()`
+    * Returns the path to builtin. Not guaranteed to return the absolute path.
```

Comment:
Plus it will likely get removed for broken once sending mods from the server is added